pkgs/servers/mautrix-telegram/default.nix

```diff
@@ -1,18 +1,28 @@
 { lib, python3, mautrix-telegram, fetchFromGitHub }:
 
-with python3.pkgs;
-
 let
+  python = python3.override {
+    packageOverrides = self: super: {
+      sqlalchemy = super.sqlalchemy.overrideAttrs (oldAttrs: rec {
+        version = "1.3.24";
```

Comment:
Please also override "name", since it will otherwise use the old version in the derivation name.

Reply:
Oops, I forgot to use `overridePythonAttrs`.